### PR TITLE
feature(virtualenv): add 'system-site-packages' option

### PR DESCRIPTION
**Pull request by vlad0337187**
# Pull Request Check List

Resolves: #issue-number-here

- [+] Added **tests** for changed code.
- [+] Updated **documentation** for changed code.

Closes: #1982
Resolves: #1393

### CODE REVIEW

**Comment by vlad0337187** [comment]
This is just a rebase to `master` for [already implemented PR](https://github.com/python-poetry/poetry/pull/1982) [for issue](https://github.com/python-poetry/poetry/issues/1393)

**Comment by meakbiyik** [comment]
Great work, this commit will have a huge effect on my SSD capacity 😄

**Comment by vlad0337187** [comment]
@abn , could you review it please when you'll have some free time ?
Thanks.

File: tests/utils/test_env.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def test_activate_activates_non_existing_virtualenv_no_envs_file(
     m.assert_called_with(
         Path(tmp_dir) / "{}-py3.7".format(venv_name),
         executable="python3.7",
-        flags={"always-copy": False},
+        flags={"always-copy": False, "system-site-packages": False},
```
Comment (abn) [inline tests/utils/test_env.py:162]: Would be great to move this to a fixture so future additions or changes to defaults would be cleaner.

**Review by abn — CHANGES_REQUESTED** [review]
Thank you @vlad0337187. The change looks good to me. I have added one comment to help with future code maintanence.

**Comment by henryiii** [comment]
This also could be useful on Apple Silicon, where there are no NumPy wheels yet but most systems (homebrew, conda-forge) provide binaries for NumPy.

**Review by abn — APPROVED** [review]

**Comment by daevski** [comment]
Thank you _all_!!! This will be very useful, I've been waiting on it ;)

**Pull request merged**